rules.d/60-persistent-storage.rules.in

```diff
@@ -59,6 +59,8 @@ KERNEL=="vd*", ATTRS{serial}=="?*", ENV{ID_SERIAL}="$attr{serial}", SYMLINK+="di
 
 # ATA
 KERNEL=="sd*[!0-9]|sr*", ENV{ID_SERIAL}!="?*", SUBSYSTEMS=="scsi", ATTRS{vendor}=="ATA", IMPORT{program}="ata_id --export $devnode"
+KERNEL=="sd*[!0-9]|sr*", ENV{ID_SERIAL}=="?*", ENV{ID_PERIPHERAL_DEV_TYPE}=="20", PROGRAM="scsi_id -u -g $devnode", \
```

Comment:
Is it necessary to call `scsi_id` here? I mean, the serials reported by ata_id and scsi_id are different??
If they are equivalent, then simply use `ID_SERIAL`.
If they are different, then why the condition `ENV{ID_SERIAL}=="?*"` is in this line?
Also, I think, it is better to add condition `ENV{ID_BUS}=="ata"`.